[2023-09-19T13:56:41.596+0545] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: project_dag.load_to_postgres_task manual__2023-09-19T08:11:36.232606+00:00 [queued]>
[2023-09-19T13:56:41.600+0545] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: project_dag.load_to_postgres_task manual__2023-09-19T08:11:36.232606+00:00 [queued]>
[2023-09-19T13:56:41.600+0545] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-19T13:56:41.610+0545] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): load_to_postgres_task> on 2023-09-19 08:11:36.232606+00:00
[2023-09-19T13:56:41.615+0545] {standard_task_runner.py:57} INFO - Started process 24763 to run task
[2023-09-19T13:56:41.619+0545] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'project_dag', 'load_to_postgres_task', 'manual__2023-09-19T08:11:36.232606+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/project_dag.py', '--cfg-path', '/tmp/tmpd7xz8g11']
[2023-09-19T13:56:41.621+0545] {standard_task_runner.py:85} INFO - Job 93: Subtask load_to_postgres_task
[2023-09-19T13:56:41.670+0545] {task_command.py:415} INFO - Running <TaskInstance: project_dag.load_to_postgres_task manual__2023-09-19T08:11:36.232606+00:00 [running]> on host ubuntu-Lenovo-Legion-5-15ARH05
[2023-09-19T13:56:41.709+0545] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='project_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_task' AIRFLOW_CTX_EXECUTION_DATE='2023-09-19T08:11:36.232606+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-19T08:11:36.232606+00:00'
[2023-09-19T13:56:41.710+0545] {sql.py:274} INFO - Executing: COPY user_input_table FROM '/tmp/user_data.csv' DELIMITER ',' CSV HEADER;
[2023-09-19T13:56:41.713+0545] {base.py:73} INFO - Using connection ID '***_conn_1' for task execution.
[2023-09-19T13:56:41.757+0545] {base.py:73} INFO - Using connection ID '***_conn_1' for task execution.
[2023-09-19T13:56:41.767+0545] {sql.py:418} INFO - Running statement: COPY user_input_table FROM '/tmp/user_data.csv' DELIMITER ',' CSV HEADER;, parameters: None
[2023-09-19T13:56:41.769+0545] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/Desktop/Airflow/venv_airflow/lib/python3.10/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/ubuntu/Desktop/Airflow/venv_airflow/lib/python3.10/site-packages/airflow/providers/common/sql/hooks/sql.py", line 392, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/ubuntu/Desktop/Airflow/venv_airflow/lib/python3.10/site-packages/airflow/providers/common/sql/hooks/sql.py", line 423, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedFile: could not open file "/tmp/user_data.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.

[2023-09-19T13:56:41.772+0545] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=project_dag, task_id=load_to_***_task, execution_date=20230919T081136, start_date=20230919T081141, end_date=20230919T081141
[2023-09-19T13:56:41.780+0545] {standard_task_runner.py:104} ERROR - Failed to execute job 93 for task load_to_***_task (could not open file "/tmp/user_data.csv" for reading: No such file or directory
HINT:  COPY FROM instructs the PostgreSQL server process to read a file. You may want a client-side facility such as psql's \copy.
; 24763)
[2023-09-19T13:56:41.789+0545] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-19T13:56:41.803+0545] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
